### ISSUE TITLE
make docker/singularity container

### ISSUE THREAD

**Issue by jdkent** [HBClab/NiBetaSeries#127]
[original comment](https://github.com/openjournals/joss-reviews/issues/1295#issuecomment-477289593)
> Would it be worth making a Singularity image for this? For example, I'm almost exclusively running fMRIPRep and MRIQC apps on a server via Singularity because I don't have installation privileges. I suppose the alternative is indicating to users to pip install nibetaseries in a conda environment or something along these lines.

**Comment by PeerHerholz** [comment]
If okay for you @jdkent and everybody else, I would tackle this, generating a `bash` script that utilizes [neurodocker](https://github.com/kaczmarj/neurodocker) to generate a Docker- and Singularityfile. I would open a respective branch on my fork and push to master, including a `README.md` that has a small stepwise tutorial on how to recreate the images. Regarding `hosting` on `docker hub` I would suggest that you @jdkent, build the images locally and then push it to your `docker hub`, eh? Once that is done, we can easily do the binder related fun addressed in #129.

**Comment by jdkent** [comment]
Thanks @PeerHerholz! That sounds like a reasonable plan of action. For now, all dependencies should be installable via pip, but this may change in the future.

One stretch goal would be to also automate the building of the container (probably through `circleci`) in an analogous way that [fmriprep](https://github.com/poldracklab/fmriprep/blob/d29902e2a1e03840dc0c0e55e9dedd0d2dc30fca/.circleci/config.yml#L4-L76) builds containers.

**Comment by PeerHerholz** [comment]
The initial draft of the images is proposed in #140.
